src/search.ts

```diff
@@ -1,14 +1,24 @@
-import { assert } from '@sindresorhus/is'
+import is, { assert } from '@sindresorhus/is'
 
 import { emojiData } from './data.js'
 import { normalizeName } from './utils.js'
 
-export const search = (keyword: string) => {
-  assert.string(keyword)
+/**
+ * Search for emojis containing the provided name or pattern in their name.
+ */
+export const search = (keyword: RegExp | string) => {
+  assert.any([is.default.string, is.default.regExp], keyword)
```

Comment:
[Non-actionable] Hmm, it is unfortunate to have to use the `.default` before every property of `is`. But we're on an older version of `@sindresorhus/is` that doesn't set up ESM defaults properly yet. I think this is reasonable for now, and a followup issue could be to bump `@sindresorhus/is` to latest.

Reply:
Oh I mean I'm not asking you to take any action in this PR. I'm not requesting changes, just noting something for the future.